shifter/theme/templates/base.html

```diff
@@ -2,79 +2,120 @@
 {% load static %}
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        {% block title %}<title>Shifter</title>{% endblock %}
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <meta http-equiv="X-UA-Compatible" content="ie=edge">
-        {% block head %}
-        {% endblock %}
-        {% tailwind_css %}
-    </head>
 
-    <body class="font-sans leading-normal tracking-normal bg-white">
-        <nav class="px-2 py-2.5">
-            <div class="container flex flex-wrap justify-between items-center mx-auto">
-                <a href="/" class="flex items-center">
-                    <img src="{% static 'img/logo.svg' %}" class="mr-3 h-6 sm:h-10" alt="Shifter Logo">
-                </a>
-                <button data-collapse-toggle="mobile-menu" type="button" class="inline-flex justify-center items-center ml-3 text-gray-400 rounded-lg md:hidden hover:text-gray-900 focus:outline-none focus:ring-2 focus:ring-blue-300" aria-controls="mobile-menu-2" aria-expanded="false">
+<head>
+    {% block title %}<title>Shifter</title>{% endblock %}
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    {% block head %}
+    {% endblock %}
+    {% tailwind_css %}
+</head>
+
+<body class="font-sans leading-normal tracking-normal bg-white">
```

Comment:
Can you change the body classes to the following:
```
<body class="flex flex-col font-sans leading-normal tracking-normal bg-white h-screen">
```

This is to make the footer appear at the bottom